api/TrackHandler.py

```diff
@@ -77,18 +87,22 @@ def addLabel(data):
     with open(rel_path, 'w') as f:
         f.writelines(file_output)
 
-    jh.addJob(data)
+    lock.release()
 
     return data
 
 
 # Removes label from label file
 def removeLabel(data):
-    rel_path = cfg.dataPath + data['name'] + '_Labels.bedGraph'
+    rel_path = '%s%s_Labels.bedGraph' % (cfg.dataPath, data['name'])
```

Comment:
this is repetitive (same path defined above), please abstract using function/class and/or use a database isntead of file system as mentioned above

Reply:
Non factor with DB Implemenation